pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -278,19 +272,16 @@ func (dswp *desiredStateOfWorldPopulator) findAndRemoveDeletedPods() {
 		}
 
 		if runningContainers {
-			klog.V(4).Infof(
-				"Pod %q still has one or more containers in the non-exited state. Therefore, it will not be removed from desired state.",
-				format.Pod(volumeToMount.Pod))
+			klog.V(4).InfoS("The pod still has one or more containers in the non-exited state. Therefore, it will not be removed from desired state.", "pod", klog.KObj(volumeToMount.Pod))
 			continue
 		}
 		exists, _, _ := dswp.actualStateOfWorld.PodExistsInVolume(volumeToMount.PodName, volumeToMount.VolumeName)
 		if !exists && podExists {
-			klog.V(4).Infof(
-				volumeToMount.GenerateMsgDetailed(fmt.Sprintf("Actual state has not yet has this volume mounted information and pod (%q) still exists in pod manager, skip removing volume from desired state",
-					format.Pod(volumeToMount.Pod)), ""))
+			klog.V(4).InfoS(volumeToMount.GenerateMsgDetailed(fmt.Sprintf("Actual state has not yet has this volume mounted information and pod (%q) still exists in pod manager, skip removing volume from desired state",
```

Comment:
Please extract all information hat volumeToMount.GenerateMsgDetailed and put it as normal keysto InfoS method.